apps/base-docs/docs/pages/wallet-app/getting-started.mdx

```diff
@@ -191,9 +191,10 @@ Yes. The Farcaster SDK supports Vue, Angular, Svelte, and any web framework.
 
 ## Technical Resources
 
-- **Live Code Examples**: [https://github.com/base/demos/tree/master/minikit](https://github.com/base/demos/tree/master/minikit)
-- **MiniKit**: [https://docs.base.org/builderkits/minikit/](https://docs.base.org/builderkits/minikit/)
-- **Mini Apps Guide**: [https://docs.base.org/wallet-app/mini-apps](https://docs.base.org/wallet-app/mini-apps)
-- **Debug Tools**: [https://farcaster.xyz/\~/developers/mini-apps/debug](https://farcaster.xyz/~/developers/mini-apps/debug)
-- **Farcaster SDK**: [https://miniapps.farcaster.xyz/docs](https://miniapps.farcaster.xyz/docs)
-- **Component Library**: [https://o](https://onchainkit.xyz/)
+- [**Live Code Examples**](https://github.com/base/demos/tree/master/minikit)
+- [**MiniKit**](https://docs.base.org/builderkits/minikit/)
+- [**Mini Apps Guide**](https://docs.base.org/wallet-app/mini-apps)
+- [**Debug Guide**](https://docs.base.org/builderkits/minikit/debugging)
+- [**Debug Tools**](https://farcaster.xyz/~/developers/mini-apps/debug)
+- [**Farcaster SDK**](https://miniapps.farcaster.xyz/docs)
```

Comment:
this is not a valid link: 
<img width="899" alt="image" src="https://github.com/user-attachments/assets/10e91a5b-dc26-4c2e-a1f2-a1413eff3df7" />